From 5791177ee20db179cc73d30834e9a2fd80dd9b57 Mon Sep 17 00:00:00 2001
From: Josh Wilsdon <jwilsdon@joyent.com>
Date: Thu, 8 Dec 2016 16:34:57 -0800
Subject: [PATCH] OS-5783 make .zonecontrol a mount from the GZ instead of a
 directory in the dataset Reviewed by: Robert Mustacchi <rm@joyent.com>
 Approved by: Josh Wilsdon <josh@wilsdon.ca>

---
 usr/src/lib/brand/lx/zone/platform.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/usr/src/lib/brand/lx/zone/platform.xml b/usr/src/lib/brand/lx/zone/platform.xml
index 3ab38dd206..5a0d757c55 100644
--- a/usr/src/lib/brand/lx/zone/platform.xml
+++ b/usr/src/lib/brand/lx/zone/platform.xml
@@ -59,6 +59,8 @@
 	    opt="ro" type="lofs" />
 	<global_mount special="/etc/zones/%z.xml"
 	    directory="/native/etc/zones/%z.xml" opt="ro" type="lofs" />
+	<global_mount special="/var/zonecontrol/%z" directory="/native/.zonecontrol"
+	    opt="ro,nodevices,nosetuid,noexec" type="lofs" />
 
 	<!-- Local filesystems to mount when booting the zone -->
 	<mount special="/native/dev" directory="/dev" type="lx_devfs" />
-- 
2.21.0

